content/docs/vmclarity/getting-started/deploy-kubernetes/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Deploy on Kubernetes
+weight: 110
+---
+
+## Prerequisites
+
+* Install a tool to run local Kubernetes clusters. Here, [Kind](https://kind.sigs.k8s.io/) is used as the default option for creating a local cluster.
```

Comment:
I'd assume the visitor of this page would like to install VMClarity on a "real" cluster

```suggestion
* You need to have a Kubernetes cluster. To try things out first, you could use [Kind](https://kind.sigs.k8s.io/) to create a local cluster.
```

Reply:
will keep the tool requirement here and add creating a cluster as a first step as suggested below

Reply:
the prerequisite is the K8s cluster itself
```suggestion
* Running Kubernetes cluster. In this example, [Kind](https://kind.sigs.k8s.io/) is used as the default option for managing a local cluster.
```